src/legacy/core_plugins/kibana/public/context/api/__tests__/successors.js

```diff
@@ -150,6 +155,7 @@ describe('context app', function () {
         'INDEX_PATTERN_ID',
         '@timestamp',
         'desc',
+        ANCHOR_TIMESTAMP_3000,
```

Comment:
Not sure whether I understand correctly, but shouldn't this be `(new Date(MS_PER_DAY * 3)).toJSON()` instead?

Reply:
it's on top of the file, you think it's better doing the conversion in every function?
`
const ANCHOR_TIMESTAMP_3000 = (new Date(MS_PER_DAY * 3000)).toJSON();
`

Reply:
Thank you for the hint, I've adapted the tests to use the right iso value everywhere